src/site/content/en/blog/autowebperf/index.md

```diff
@@ -6,6 +6,11 @@ description: >
   A new modular tool that enables automatic gathering of performance data form multiple sources.
 date: 2020-12-09
 # updated: 2020-07-22
+authors:
+  - jonchen
+  - gilbertococchi
+  - antoinebisch
+hero: hero.png
```

Comment:
@jonchenn Please optimize the hero image, I suggest switching to JPG. Check out our guidelines: https://web.dev/handbook/markup-media/#optimization

Reply:
Update images with JPEGs.